templates/vue/src/components/modals/NodeModal/index.vue

```diff
@@ -921,18 +905,30 @@ export default {
         if (!validPreviousAnswers) {
           errMsgs.push("Please select a previous activity to display")
         }
+        if (this.node.typeData.activity.questions[0].answerTypes.list.enabled) {
+          let listQuestion = this.node.typeData.activity.questions[0].answerTypes
+            .list
+          if (listQuestion.minFields < 1) {
+            errMsgs.push("Minimum # of fields for list activity must be >= 1")
+          }
+          if (listQuestion.maxFields.enabled) {
+            if (listQuestion.maxFields.value < listQuestion.minFields) {
+              errMsgs.push(
+                "Maximum # of fields for list activity must be >= minumum # fields"
+              )
```

Comment:
Similarly, there should be a check for if minvalue > maxvalue

Reply:
That check covers the logic of `minvalue > maxvalue`